src/DocumentFormat.OpenXml/Packaging/ImagePartTypeExtensions.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+
+namespace DocumentFormat.OpenXml.Packaging
+{
+    /// <summary>
+    /// Provides extensions to working with ImagePart and ImagePartType
+    /// </summary>
+    public static class ImagePartExtensions
+    {
+        /// <summary>
+        /// Extends an overloaded AddImagePart method that
```

Comment:
can you use `<see cref="..."/>` tags where there actual types/methods?

Reply:
Feedback handled in 828ffa141a9684d12e63e719ff4c013ac56f435c. Additionally, cleared all build warnings. Thanks for helping me get this straight!